bricks/stm32/main.c

```diff
@@ -157,6 +225,66 @@ static void run_user_program(uint32_t len, uint8_t *buf, bool run_repl) {
     pbsys_user_program_unprepare();
 }
 
+// MPY info and data for one script or module.
+typedef struct {
+    /**
+     * Size of the mpy program.
+     */
+    uint32_t mpy_size;
+    /**
+     * Null-terminated name of the script, padded with additional zeros to
+     * make the size an integer multiple of 4.
+     */
+    char mpy_name[];
```

Comment:
Should there be an upper bound on file name size? Or maybe would be simpler to just use a fixed size? Or use one byte to store the string length?

Reply:
Is this stored the name `module.mpy` or just `module`?